src/super_gradients/training/dataloaders/dataloader_factory.py

```diff
@@ -412,3 +416,21 @@ def supervisely_persons_val(dataset_params: Dict = {}, dataloader_params: Dict =
                            train=False,
                            dataset_params=dataset_params,
                            dataloader_params=dataloader_params)
+
+
+def pascal_voc_train(dataset_params: Dict = {}, dataloader_params: Dict = {}):
```

Comment:
maybe call it pascal_voc_detection_train and pascal_voc_detection_val?

Reply:
sure, renamed in https://github.com/Deci-AI/super-gradients/pull/332/commits/3cb87ded44227afef53deb1815d0ed8f2daf9f8f